com.unity.render-pipelines.universal/Shaders/Nature/SpeedTree7CommonPasses.hlsl

```diff
@@ -1,6 +1,10 @@
 #ifndef UNIVERSAL_SPEEDTREE7COMMON_PASSES_INCLUDED
 #define UNIVERSAL_SPEEDTREE7COMMON_PASSES_INCLUDED
 
+// Disable warnings we aren't interested in
+#pragma warning (disable : 3571) // "pow(f,e) will not work for negative f"; however in majority of our calls to pow we know f is not negative
+#pragma warning (disable : 3206) // implicit truncation of vector type
```

Comment:
This is very dangerous, it might silent fail issues. Where is the truncation happening? we should probably fix the truncation.

Reply:
Found the culprit and fixed it.